backend/services/implementations/classService.ts

```diff
@@ -0,0 +1,260 @@
+import { UserDTO } from "../../types";
+import MgClass, { Class } from "../../models/class.model";
+import { getErrorMessage } from "../../utilities/errorUtils";
+import logger from "../../utilities/logger";
+import {
+  IClassService,
+  ClassRequestDTO,
+  ClassResponseDTO,
+  StudentResponseDTO,
+  StudentRequestDTO,
+} from "../interfaces/classService";
+import IUserService from "../interfaces/userService";
+import {
+  ITestSessionService,
+  TestSessionResponseDTO,
+} from "../interfaces/testSessionService";
+
+const Logger = logger(__filename);
+
+class ClassService implements IClassService {
+  userService: IUserService;
+
```

Comment:
```suggestion
```

Reply:
I don't know why but the linter keeps on adding an empty line

Reply:
ah lol ok just leave as is